analysis/plotconfigs/svt/Svt2DBl.json

```diff
@@ -8,20 +8,11 @@
 
         },
 
-    "svtHitOccupancy_h" : {
-            "xtitle" : "# hits per channel",
-            "bins" : 100000,
-            "minX" : 0,
-            "maxX" : 100000,
-            "ytitle" : "Events"
-
-        },
-
     "SvtHybridsHitN_h" : {
             "xtitle" : "# of Strips",
             "bins" : 10000,
             "minX" : 0,
-            "maxX" : 10000,
+            "maxX" : 200,
```

Comment:
this should be higher than 200, but also there is no reason to have more bins than integers on the x axis. make this something like 2000 bins from -0.5 to 1999.5

Reply:
Done